### ISSUE TITLE
deleted colors

### ISSUE THREAD

**Issue by kubiqsk** [padaliyajay/php-autoprefixer#22]
when there are expressions like
`background-color: rgba(var(--bs-primary-rgb), var(--bs-bg-opacity)) !important;`
or
`color: rgba(var(--bs-body-color-rgb), var(--bs-text-opacity)) !important;`
then it will complete delete them

**Comment by Ruud68** [comment]
Okay, just had a look at this one.
This is not an issue with this repo but an issue with the sabberworm repo that this repo uses.

there is a fix that will be in the latest release (not released yet): https://github.com/sabberworm/PHP-CSS-Parser/issues/257

**Comment by kubiqsk** [comment]
I did output also compiled CSS before autoprefixing and it contains all needed expressions
https://scss.kubiq.sk/style-unprefixed.css
so i'm pretty sure, it's actually issue in here

**Comment by kubiqsk** [comment]
after that i just take this and
```
$autoprefixer = new Autoprefixer( $unprefixed_css );
$prefixed_css = $autoprefixer->compile();
file_put_contents( 'style.css', $prefixed_css );
```
and after that, there are no more these values
https://scss.kubiq.sk/style.css

**Comment by padaliyajay** [comment]
he mean "[PHP-CSS-Parser](https://github.com/sabberworm/PHP-CSS-Parser)" repo, and not SCSS repo. @kubiqsk

**Comment by kubiqsk** [comment]
you're right! Sorry
But it's more than 6 months old
and they say it's solved in 8.3.1 (which is not officially released?)

**Comment by kubiqsk** [comment]
so yes, when I download ZIP of https://github.com/sabberworm/PHP-CSS-Parser and replace it in `/vendor/sabberworm/php-css-parser/lib/Sabberworm/CSS/` then everything works

**Comment by Ruud68** [comment]
I thing that the sabberworm repo is not yet made available (version 8.4), not sure what the status / delay is.
While testing this I found also other issue reported here: https://github.com/sabberworm/PHP-CSS-Parser/issues/357
for BS5 css that is no issue, but when you apply codestyle and break long lines onto multiple lines then you might have an issue.
